frontend/src/main/webapp/pagevisit.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Place Visit</title>
+     <script src="pagevisit.js"></script>
+     <style>
+         ul{
+             border: 1px solid black;
+         }
+     </style>
+  </head>
+  <body onload ="PageData()">
+    <h1>Trip Me Out!</h1>
+    <br/><br/>
+    <div >
```

Comment:
I think you accidentally left another space here and also in the div with the image